diktat-rules/src/test/resources/test/paragraph2/kdoc/ConstructorCommentExpected.kt

```diff
@@ -0,0 +1,56 @@
+package test.paragraph2.kdoc
+
+/**
+ * Simple class
+ * @property name  should replace
+*/
+class A constructor(
+        val name: String
+)
+
+/**
+ * @property age age
+ * @property lastName Ivanov
+*/
+class A(
+        val age: Int,
+        val lastName: String
+) {
+
+}
+
+class Out{
+    /**
+     * Inner class
+     * @property name Jack
+*/
+    class In(
+            val name: String
+    ){}
+}
+
+/**
+ *
+ * some text
+*/
+class A (
+        var name: String
+){}
+
+/**
+ * @property name another text
+ * some text
+*/
+class A (
+        var name: String
+){}
+
+/**
+ * @property name another text
+ */
+class A (
```

Comment:
Do you have this example in warn test too? We should get unfixable warning in this case, right?

Reply:
No, I don't have, already added. Yes, it should be unfixable